snippets/ponylang-mode/interface

```diff
@@ -3,4 +3,6 @@
 # name: interface
 # contributor: Sean T. Allen <sean@monkeysnatchbanana.com>
 # --
-interface ${1:name}$0
\ No newline at end of file
+interface ${1:name}
```

Comment:
this seems fairly opinionated. when you could do interface snippet then fun snippet.